pyimzml/ImzMLParser.py

```diff
@@ -293,6 +293,9 @@ def get_spectrum_as_string(self, index):
         intensity_string = self.m.read(lengths[1])
         return mz_string, intensity_string
 
+    def getionimage(self, mz_value, tol=0.1, z=None, reduce_func=sum):
+        getionimage(self, mz_value, tol, z, reduce_func)
+
```

Comment:
I find the redundance rather confusing. From a conceptual point of view, making an ion image is not a duty of the parser; it's just a tool for reading a file. Concerns are better separated if getionimage is not an instance method